File: yoroflow-ui/src/app/engine-module/process-instance-task-list/pagination-vo.ts
export class PaginationVO {
    index: number;
    size: number;
    direction: string;
    columnName: string;
    processInstanceId: string;
    taskStatus: string;
    filterValue: FilterValueVO[] = [];
}
export class FilterValueVO {
    filterIdColumn: string;
    operators: string;
    filterIdColumnValue: string;
}

File: support-objects/src/main/java/com/yorosis/yoroapps/grid/vo/PaginationVO.java
package com.yorosis.yoroapps.grid.vo;

import java.util.UUID;

import com.yorosis.yoroapps.vo.FilterValueVO;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class PaginationVO {
	private String gridId;
	private int index;
	private int size;
	private String direction;
	private String columnName;
	private FilterValueVO[] filterValue;
	private Boolean isNoRoles;
	private Boolean isUnAssigned;
	private UUID id;
}
